### PR TITLE
Replace instances of XCode with Xcode which is the proper casing for the name.

### PR DESCRIPTION
**Pull request by sergiocampama**

### CODE REVIEW

**Comment by sergiocampama** [comment]
how can I how the tests failed? there seems to be no logs

File: third_party/zlib/contrib/amd64/amd64-match.S [hunk 1]
```diff
@@ -92,7 +92,7 @@ printf("#define dsNiceMatch     (%3u)(%%rdi)\n",(int)(((char*)&(s->nice_match))-
 
 
 /*
-  to compile for XCode 3.2 on MacOSX x86_64
+  to compile for Xcode 3.2 on MacOSX x86_64
```
Comment (aragos) [inline third_party/zlib/contrib/amd64/amd64-match.S]: This is a third_party file. I don't think it makes sense to fix spelling in it.

File: third_party/zlib/contrib/gcc_gvmat64/gvmat64.S [hunk 2]
```diff
@@ -1,12 +1,12 @@
 /*
 ;uInt longest_match_x64(
 ;    deflate_state *s,
-;    IPos cur_match);                             // current match 
+;    IPos cur_match);                             // current match
```
Comment (aragos) [inline third_party/zlib/contrib/gcc_gvmat64/gvmat64.S]: Please also revert this thirds party file.

File: third_party/zlib/contrib/amd64/amd64-match.S (hunk 1 above)
Comment (sergiocampama) [reply third_party/zlib/contrib/amd64/amd64-match.S]: Reverted all third_party files

Comment (sergiocampama) [reply third_party/zlib/contrib/amd64/amd64-match.S]: Reverted all third_party files

File: third_party/zlib/contrib/gcc_gvmat64/gvmat64.S (hunk 2 above)
Comment (sergiocampama) [reply third_party/zlib/contrib/gcc_gvmat64/gvmat64.S]: Reverted all third_party files.

**Pull request closed without merging**